docs/component.md

```diff
@@ -273,6 +273,36 @@ Changes `<hello-world>Hi There</hello-world>` into:
 <hello-world><h1>Hi There</h1></hello-world>
 ```
 
+Essentially, the children of the component tag will be treated as it's [can-component/content], 
+to be rendered wherever the tag is provided in the component view.
+
+If no view is provided to a Component, it will render it's content naively.
```

Comment:
Are you trying to say:

> If no LIGHT DOM is provided to a component, it will render the <content> element's LIGHT DOM?

We don't call the LIGHT DOM a `view`.  The `view` is the component's `view` property.  

By `natively` it sounds to me like something the BROWSER apis would just do.

Also `it's` => `its`

Reply:
I think maybe it's more accurate to say if there is no view property on a Component, that Component will simply render its LIGHT_DOM